### PR TITLE
ecma_make_boolean_value(bool) function is added

### PR DESCRIPTION
**Pull request by ktorpi**
ecma_make_boolean_value(bool) function is added in order to make ecma_value form a raw bool.

JerryScript-DCO-1.0-Signed-off-by: István Kádár ikadar@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/base/ecma-helpers-value.c [hunk 1]
```diff
@@ -336,6 +336,16 @@ ecma_make_simple_value (const ecma_simple_value_t simple_value) /**< simple valu
 } /* ecma_make_simple_value */
 
 /**
+ * Cretes an ecma value from the given raw boolean.
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers-value.c]: @ return comment missing

File: jerry-core/ecma/base/ecma-helpers.h [hunk 2]
```diff
@@ -132,6 +132,7 @@ extern bool ecma_is_value_object (ecma_value_t) __attr_pure___;
 extern void ecma_check_value_type_is_spec_defined (ecma_value_t);
 
 extern ecma_value_t ecma_make_simple_value (const ecma_simple_value_t value) __attr_const___;
+extern ecma_value_t ecma_make_boolean_value (const bool boolean_value) __attr_const___;
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers.h]: (bool) is enough
const is not needed for primitive types, and name is not used as well.

**Comment by zherczeg** [comment]
Just a few fixes for the new function.

File: jerry-core/ecma/base/ecma-helpers-value.c (hunk 1 above)
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers-value.c]: 'Cretes' -> Creates

File: jerry-core/ecma/base/ecma-helpers-value.c [hunk 3]
```diff
@@ -336,6 +336,18 @@ ecma_make_simple_value (const ecma_simple_value_t simple_value) /**< simple valu
 } /* ecma_make_simple_value */
 
 /**
+ * Creates an ecma value from the given raw boolean.
+ *
+ * @return boolean ecma_value
+ */
+inline ecma_value_t __attr_const___ __attr_always_inline___
+ecma_make_boolean_value (const bool boolean_value) /**< raw bool value from which the ecma value will be created */
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers-value.c]: This const is not needed. Interesting that the compiler doesn't complain.

LGTM after that.

**Comment by LaszloLango** [comment]
LGTM

**Comment by zherczeg** [comment]
Great. Still LGTM.

**Pull request merged**